portal-api/src/v1/services/utilisation-report/email-service.ts

```diff
@@ -0,0 +1,82 @@
+import { array, string } from 'zod';
+import { PaymentOfficerTeam, asString } from '@ukef/dtfs2-common';
+import sendEmail from '../../email';
+import { EMAIL_TEMPLATE_IDS } from '../../../constants';
+import { formatDateForEmail } from '../../helpers/formatDateForEmail';
+import api from '../../api';
+import { InvalidEnvironmentVariableError } from '../../errors';
+
+const getUkefGefReportingEmailRecipients = () => {
+  const EmailsSchema = array(string());
+
+  try {
+    const emails =  EmailsSchema.parse(JSON.parse(asString(process.env.UKEF_GEF_REPORTING_EMAIL_RECIPIENT, 'UKEF_GEF_REPORTING_EMAIL_RECIPIENT')));
+    return emails;
+  } catch (error) {
+    console.error('Failed to parse UKEF_GEF_REPORTING_EMAIL_RECIPIENTS ', error);
+    throw new InvalidEnvironmentVariableError('Failed to parse UKEF_GEF_REPORTING_EMAIL_RECIPIENTS');
+  }
+};
+
+/**
+ * Sends notification email to UKEF GEF reporting email recipients that a utilisation report has been submitted
+ * @param {string} bankName - name of the bank
+ * @param {string} reportPeriod - period for which the report covers as a string, eg. June 2023
```

Comment:
That was pre-existing, we get the formatted report period in the request from portal so that has been used I assume for simplicity over reformatting the report period object?

Reply:
That makes sense to me